src/frontend/packages/lib_video/src/components/common/DashboardControlPane/DashboardTabStatistics/index.spec.tsx

```diff
@@ -31,4 +31,44 @@ describe('DashboardTabStatistics', () => {
     expect(await screen.findByText('Viewers')).toBeInTheDocument();
     expect(screen.getByText(3)).toBeInTheDocument();
   });
+
+  it('displays one viewer', async () => {
+    const video = videoMockFactory();
+
+    const deferred = new Deferred();
+
+    fetchMock.mock(`/api/videos/${video.id}/stats/`, deferred.promise);
+
+    render(wrapInVideo(<DashboardTabStatistics />, video));
+
+    // Loader
+    expect(screen.getByRole('status')).toBeInTheDocument();
+
+    deferred.resolve({
+      nb_views: 1,
+    });
+
+    expect(await screen.findByText('Viewer')).toBeInTheDocument();
+    expect(screen.getByText(1)).toBeInTheDocument();
+  });
+
+  it('displays no viewer', async () => {
+    const video = videoMockFactory();
+
+    const deferred = new Deferred();
+
+    fetchMock.mock(`/api/videos/${video.id}/stats/`, deferred.promise);
+
+    render(wrapInVideo(<DashboardTabStatistics />, video));
+
+    // Loader
+    expect(screen.getByRole('status')).toBeInTheDocument();
+
+    deferred.resolve({
+      nb_views: 0,
+    });
+
+    expect(await screen.findByText('Viewer')).toBeInTheDocument();
+    expect(screen.getByText(0)).toBeInTheDocument();
```

Comment:
I think we should display `0 viewers`

Reply:
why ?

Reply:
https://ell.stackexchange.com/questions/115008/singular-or-plural-with-zero-the-temperature-will-drop-to-zero-degrees

Reply:
Awesome, thanks !